### PR TITLE
version 0.2

### PR DESCRIPTION
**Pull request by AlbertDeFusco**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,12 +23,14 @@ requirements:
     - pip
   run:
     - python >=3.8
-    - anaconda-cli-base
-    - anaconda-cloud-auth
+    - anaconda-cli-base >=0.2
```
Comment (mattkram) [inline recipe/meta.yaml:26]: Pins LGTM. We will need the actual Sha from the package. Assuming waiting for the real or from the autotick bot.

**Comment by AlbertDeFusco** [comment]
@conda-forge-admin, please rerender

**Comment by mattkram** [comment]
@conda-forge-admin, please rerender

**Review by mattkram — APPROVED** [review]
:rocket:

**Pull request merged**